test/bin/stress_testing.sh

```diff
@@ -25,12 +25,46 @@ function run {
 
   if [ "${SSH_HOSTNAME:-}" ]; then
     # shellcheck disable=SC2086
-    ssh -o LogLevel=ERROR ${SSH_PORT_PARAM:-} ${SSH_PKEY_PARAM:-} ${SSH_USER_PARAM:-}${SSH_HOSTNAME} "$@"
+    ssh -o StrictHostKeyChecking=no -o LogLevel=ERROR ${SSH_PORT_PARAM:-} ${SSH_PKEY_PARAM:-} ${SSH_USER_PARAM:-}${SSH_HOSTNAME} "$@"
   else
     "$@"
   fi
 }
 
+# install dependecies
+function install_kernel-modules-extra {
+  if ( ! run rpm-ostree status &> /dev/null); then
+    if ( ! run rpm -q kernel-modules-extra &> /dev/null); then
+      run sudo dnf -y install kernel-modules-extra &> /dev/null
+      
+      run sudo reboot &
+      sleep 10
+      TIMEOUT_LIMIT=$(( SECONDS + 300 ))
+      until run true || (( SECONDS > TIMEOUT_LIMIT )); do
+          sleep 10
+      done
```

Comment:
There is a robust way to check if a host was rebooted. 
Please, see how it's implemented in the test and let's do it a in similar way here.
https://github.com/openshift/microshift/blob/main/test/resources/microshift-host.resource#L49

Reply:
Thanks, but I don't think I need to add a robust validation here. I just need to wait until the VM is up again.

Reply:
The current code has a high chance of detecting a `false` reboot. The thing is that it's hard to predict how much time it will take a host to go down. While it's going down, ssh still works, so your `run true` after a 10s sleep has a high chance of succeeding on a host before the reboot.